x-pack/plugins/maps/public/components/_index.scss

```diff
@@ -1,5 +1,5 @@
 @import './gis_map/gis_map';
-@import './layer_addpanel/layer_addpanel';
+@import './layer_addpanel/source_select';
```

Comment:
You'll still need the `/index` here. SASS doesn't know that `index` should be the default import.